### DISCUSSION TITLE
New to Git

### DISCUSSION THREAD

**Discussion by OneMonkeyFist** [community/community#63019] [New to GitHub]
Just wanted to get my profile on the radar - and say "hi" to the community. 

I am a Disabled Veteran with cognitive challenges and am really looking to leverage ML and LLM's as a Second Brain for myself and other Veterans dealing with cognitive challenges from their service related injuries.

I am starting a new charity called Operation MonkeyFist and posted a [discussion](https://github.com/orgs/community/discussions/63016) looking for community members interested in helping out. So if anyone thinks they could bring some value or just wants a real world project to learn from - jump over to the discussion and let me know.

Thanks!!

**Comment by queenofcorgis** [comment]
Hi there, 

Thanks for making sure this gets on folks radar. We typically don't allow posts where you're just linking to another post to keep our space clutter free, but I've made sure to pass along your other post (which can remain)  to our Machine Learning team. Thank you for understanding and good luck with your project!
